File: InterviewQuestions/Python Interview Programming Questions/task12.py
# Write a Python program to sort the list of numbers

number_list = [23,134,42,43,2,56,7,34,2,34,5,67,6,56,67,456,87,798,862]

sorted_list = []

while len(number_list) > 0:

    minimum_number = min(number_list)

    min_number_index = number_list.index(minimum_number)

    number_list.pop(min_number_index)

    sorted_list.append(minimum_number)

print(sorted_list)

File: InterviewQuestions/Javascript Interview Programming Questions/task12.js
// Write a Javascript program to sort the array of numbers

let number_array = [23,134,42,43,2,56,7,34,2,34,5,67,6,56,67,456,87,798,862]

let sorted_array = []

while (number_array.length > 0) {

    minimum_number = Math.min.apply(null, number_array)

    min_number_index = number_array.indexOf(minimum_number)

    number_array.splice(min_number_index, 1)

    sorted_array.push(minimum_number)
}

console.log(sorted_array);